### ISSUE TITLE
Idea - Copy day to dates

### ISSUE THREAD

**Issue by jenstornell** [molocLab/kirby-calendar-board#17]
This is an addition to your solution for bulk tasks. Maybe you like it or maybe not, but I write it anyway.

When in a day when filled the structure field with data for a specific day, then the user want to copy this day into other days. These days are not in a row but spread out the calendar.

A possible solution would be to add another structure field that's called "Copy to date". In the structure field there would only be a date field.

It would result in a list of dates if it's used. On save, copy the day data to the dates.

I still really like your solution with bulk days so I see this as an addition to that, in case it's added.

**Comment by molocLab** [comment]
Copying an event to another date makes sense, copy a whole day with all its events not (according to me).

**Comment by jenstornell** [comment]
Yes. I had some bad luck when I was thinking yesterday. You are right of course.

That add a big problem to this idea. In the event structurefield it cannot be another structurefield så it would not be possible to repeatably add a number of dates. Instead it would require some custom field or js solution.

**Comment by molocLab** [comment]
100% custom field
